config/environments/development.rb

```diff
@@ -38,4 +39,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.x.saml_issuer = 'urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost-rails'
+  config.x.idp_url = 'http://localhost:3000/api/saml/auth'
+  config.x.idp_cert_fingerprint = 'E6:12:DC:4D:93:66:02:E0:74:AE:B8:2C:F8:2C:3A:C5:AA:20:3D:60'
 end
```

Comment:
Once in `secrets.yml`, they can be accessed like this: `Rails.application.secrets.saml_issuer`.